src/EnergyPlus/DaylightingManager.cc

```diff
@@ -6436,20 +6434,22 @@ namespace DaylightingManager {
         //  zone/window daylighting properties.
         if (DayltgInteriorIllum_firstTime) {
             int const d1(max(maxval(Zone, &ZoneData::NumSubSurfaces), maxval(ZoneDaylight, &ZoneDaylightCalc::NumOfDayltgExtWins)));
-            tmpIllumFromWinAtRefPt.allocate(d1, 2, NREFPT);
-            tmpBackLumFromWinAtRefPt.allocate(d1, 2, NREFPT);
-            tmpSourceLumFromWinAtRefPt.allocate(d1, 2, NREFPT);
+            tmpIllumFromWinAtRefPt.allocate(d1, 2, maxNumRefPtInAnyZone);
+            tmpBackLumFromWinAtRefPt.allocate(d1, 2, maxNumRefPtInAnyZone);
+            tmpSourceLumFromWinAtRefPt.allocate(d1, 2, maxNumRefPtInAnyZone);
```

Comment:
Major change is here where tmp arrays are allocated to max number of reference points instead of number of reference points from the first zone called.

Reply:
This makes good sense.  Good catch.